flake.nix

```
@@ -31,6 +31,15 @@
                       ./agimus_controller
                     ];
                   };
+                  # Remove when the CI passes. And update https://github.com/Gepetto/nix
+                  nativeCheckInputs = [ prev.python3Packages.pytest ];
+                  doCheck = true;
+                  preCheck = ''
+                    cd $out
+                  '';
+                  checkPhase = ''
+                    pytest ./tests
+                  '';
```

```suggestion
                  # Remove when the CI passes. And update https://github.com/Gepetto/nix
                  nativeCheckInputs = [ final.python3Packages.pytestCheckHook ];
```

Can you check with this ?